script/brew-install-for-cmake.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#====================================
+# @file   : brew-install-for-cmake.sh
+# @brief  : installs a package with Homebrew, keg-only, then registers that package's
+#           include, lib, bin, etc. directories for use by CMake
+# @usage  : script/brew-install-for-cmake.sh <package>
+# @param  : first, the brew formula name, optionally including a version suffix
+#           second, a truthy value indicating whether or not to leave it linked
+#====================================
+# Copyright (C) 2023 Stephen G. Tuggy and other Vega Strike contributors
+#
+# This file is part of Vega Strike.
+#
+# Vega Strike is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Vega Strike is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Vega Strike.  If not, see <https://www.gnu.org/licenses/>.
+
+set -e
+
+PACKAGE_SPEC="$1"
+LEAVE_LINKED="$2"
```

Comment:
Would be good to have some comments describing how to use this script and the meaning of the parameters.

Reply:
@BenjamenMeyer Agreed. It would be good. :-)

Reply:
... To finish the thought: Can we delay adding the comments until a future PR? Or do they need to be added right now, in your opinion?

Reply:
We can delay

Reply:
Okay, sounds good. I'll file a new issue for that documentation work